### PR TITLE
Add aliases for easy get current IPs

### PR DESCRIPTION
**Pull request by frani**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Comment by frani** [comment]
add two simple aliases, I will add more in future

**Review by TheTechRobo — APPROVED** [review]
works fine for me :D 

image

**Comment by TheTechRobo** [comment]
would suggest rebasing tho, i.e. removing some of the merge commits.

File: plugins/ips/ips.plugin.zsh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# easy get ip - internal & external
+alias ipe="curl ipinfo.io/ip"
```
Comment (cmalard) [inline plugins/ips/ips.plugin.zsh:2]:
```suggestion
alias ipe="curl -w '\n' ipinfo.io/ip"
```
[to remove](https://stackoverflow.com/questions/29497038/why-does-a-curl-request-return-a-percent-sign-with-every-request-in-zsh) the `%` at the end of the display :beers:

File: plugins/ips/ips.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+alias ipi="ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*'| grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'"
```
Comment (cmalard) [inline plugins/ips/ips.plugin.zsh:3]: What about merging the aliases ?
```suggestion
alias ip="curl -w '\n' ipinfo.io/ip && ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*'| grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'"
```
Or adding a new one
```suggestion
alias ipe="curl ipinfo.io/ip"
alias ipi="ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*'| grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1'"
alias ip="ipe && ipi"
```

Comment (frani) [reply plugins/ips/ips.plugin.zsh:3]: thanks  @cmalard but "ip" its a command in Linux, that why I avoided use it

**Comment by JuniorJPDJ** [comment]
Why closed?

**Comment by frani** [comment]
I didn't. Think so when I revert many commits ( merge commits ), and push it on my repo, gh automatically closed. So I did a new PR

**Pull request closed without merging**